tests/models/mamba2/test_modeling_mamba2.py

```diff
@@ -280,6 +335,31 @@ def setUp(self):
 
     def test_config(self):
         self.config_tester.run_common_tests()
+    
+    @require_torch_multi_gpu
+    def test_multi_gpu_data_parallel_forward(self):
+        config, inputs_dict = self.model_tester.prepare_config_and_inputs_for_common()
+
+        # some params shouldn't be scattered by nn.DataParallel
+        # so just remove them if they are present.
+        blacklist_non_batched_params = ["cache_params"]
+        for k in blacklist_non_batched_params:
+            inputs_dict.pop(k, None)
+
+        # move input tensors to cuda:O
+        for k, v in inputs_dict.items():
+            if torch.is_tensor(v):
+                inputs_dict[k] = v.to(0)
+
+        for model_class in self.all_model_classes:
+            model = model_class(config=config)
+            model.to(0)
+            model.eval()
+
+            # Wrap model in nn.DataParallel
+            model = torch.nn.DataParallel(model)
+            with torch.no_grad():
+                _ = model(**self._prepare_for_class(inputs_dict, model_class))
```

Comment:
Shouldn't be in here and is not related to the PR. We can take a look into this later.

Reply:
Alright, replaced it by a TODO.

Reply:
Same as in the other comment, let's skip it for now and mark it with a TODO.